operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -67,6 +68,56 @@
 @SuppressWarnings("checkstyle:CyclomaticComplexity")
 public abstract class Ca {
 
+    /**
+     * A certificate entry in a ConfigMap. Each entry contains an entry name and data.
+     */
+    public enum SecretEntry {
```

Comment:
What is the value we get here from using Enum instead of just constants?

Reply:
An Enum has two added benefits:
- It acts as a set of the possible keys in the secret. 
- It has methods which encapsulate the logic of the mapping of the secret key names to the broker.